tests/transformers/models/test_causal_lm_models.py

```diff
@@ -69,15 +106,19 @@ def test_causal_lm_pytorch_vs_kv_vs_ort_vs_ai100(model_name):
 
     pytorch_hf_tokens = api_runner.run_hf_model_on_pytorch(model_hf)
 
-    qeff_model = QEFFAutoModelForCausalLM(model_hf, f"{model_name}")
+    qeff_model = QEFFAutoModelForCausalLM(model_hf)
 
     pytorch_kv_tokens = api_runner.run_kv_model_on_pytorch(qeff_model.model)
 
     assert (
         pytorch_hf_tokens == pytorch_kv_tokens
     ).all(), "Tokens don't match for HF PyTorch model output and KV PyTorch model output"
 
-    onnx_model_path = qeff_model.export()
+    _, onnx_model_path = qualcomm_efficient_converter(
+        model_name=model_name,
+        model_kv=qeff_model,
+        tokenizer=tokenizer,
+    )
```

Comment:
The idea is to test the export API? why change this?

Reply:
We essentially have 2 different API's now. One is `QEfficient.export()`, other is `QEFFAutoModel.export()`. The two are not interchangeable with each other. This test was essentially testing the old (first) export API. We're replacing the `model.export()` with new implementation. So replacing the old test with old function call.

Reply:
But we are not testing the new implementation of export, compile. What I mean by that is we are not checking if onnx output with new API matches pytorch API and same for qpc. 
I am planning to make this test use the new APIs and add a test to make sure the new and old API produces same files for both onnx export and compilation.